lib/util/config-types.ts

```diff
@@ -317,6 +317,8 @@ export interface TransitOperatorConfig extends TransitOperator {
 export interface RouteViewerConfig {
   /** Whether to hide the route linear shape inside a flex zone of that route. */
   hideRouteShapesWithinFlexZones?: boolean
+  /** Remove vehiles from the map if they haven't sent an update in a number of seconds */
```

Comment:
codespell is slacking

Reply:
for every job AI steals from us, we steal one from AI